package.json

```diff
@@ -73,6 +73,7 @@
     "@nuxtjs/apollo": "^4.0.1-rc.5",
     "@nuxtjs/i18n": "^7.3.0",
     "@nuxtjs/sentry": "^6.0.1",
+    "@oruga-ui/oruga": "^0.5.8",
```

Comment:
wouldn't it be `@oruga-ui/oruga-next`?

Reply:
Yep made some test and oruga-next wasn't working with our current setup.
But there is still something that I'm missing out, if we integrate oruga to histoire then we won't need the dependency in the root of the project?